### ISSUE TITLE
Generated googlevideo link doesn't work. 

### ISSUE THREAD

**Issue by report-bugs** [yt-dlp/yt-dlp#7866]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting that yt-dlp is broken on a **supported** site
- [X] I've verified that I'm running yt-dlp version **2023.07.06** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

Germany

### Provide a description that is worded well enough to be understood

Youtube video works with my localization. I played it by Google site. I run yt-dlp with options: 
yt-dlp -f 18 https://www.youtube.com/watch?v=4CtBuDfO4Vc
yt-dlp https://www.youtube.com/watch?v=4CtBuDfO4Vc

And it doesn't work. The generated link googlevideo.com also doesn't work via browser (but original link does). 

I have tried many times. It doesn't work today and yesterday. 

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [ ] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
[debug] Command-line config: ['-vU', 'https://www.youtube.com/watch?v=4CtBuDfO4Vc']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version 2023.02.17 [a0a7c01] (linux_dir)
[debug] Python 3.10.12 (CPython x86_64 64bit) - Linux-5.4.0-156-generic-x86_64-with-glibc2.31 (OpenSSL 3.1.1 30 May 2023, glibc 2.31)
[debug] exe versions: ffmpeg 4.2.7, ffprobe 4.2.7, rtmpdump 2.4
[debug] Optional libraries: Cryptodome-3.16.0, brotli-1.0.9, certifi-2022.12.07, mutagen-1.46.0, sqlite3-2.6.0, websockets-10.4
[debug] Proxy map: {}
[debug] Loaded 1782 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Latest version: 2023.07.06, Current version: 2023.02.17
[debug] Downloading _update_spec from https://github.com/yt-dlp/yt-dlp/releases/download/2023.07.06/_update_spec
ERROR: Auto-update is not supported for unpackaged Linux executable; Re-download the latest release
[youtube] Extracting URL: https://www.youtube.com/watch?v=4CtBuDfO4Vc
[youtube] 4CtBuDfO4Vc: Downloading webpage
WARNING: [youtube] Unable to download webpage: HTTP Error 429: Too Many Requests
[youtube] 4CtBuDfO4Vc: Downloading android player API JSON
[youtube] 4CtBuDfO4Vc: Downloading iframe API JS
WARNING: [youtube] Unable to download webpage: HTTP Error 429: Too Many Requests
[youtube] 4CtBuDfO4Vc: Downloading web player API JSON
[youtube] 4CtBuDfO4Vc: Downloading initial data API JSON
[debug] Sort order given by extractor: quality, res, fps, hdr:12, source, vcodec:vp9.2, channels, acodec, lang, proto
[debug] Formats sorted by: hasvid, ie_pref, quality, res, fps, hdr:12(7), source, vcodec:vp9.2(10), channels, acodec, lang, proto, filesize, fs_approx, tbr, vbr, abr, asr, vext, aext, hasaud, id
[debug] Default format spec: bestvideo*+bestaudio/best
[info] 4CtBuDfO4Vc: Downloading 1 format(s): 248+251
[debug] Invoking http downloader on "https://rr2---sn-ntnxax8xo-cxge.googlevideo.com/videoplayback?expire=1692228953&ei=-QjdZK2PCpa4xN8PlKCC0Aw&ip=185.220.101.66&id=o-ADMT1a3K21qpg2Yfv-Ch_EtevKyZwtiiEi5F5Otlk0AX&itag=248&source=youtube&requiressl=yes&mh=2h&mm=31%2C29&mn=sn-ntnxax8xo-cxge%2Csn-4g5lznl6&ms=au%2Crdu&mv=m&mvi=2&pcm2cms=yes&pl=24&pcm2=yes&initcwndbps=642500&spc=UWF9f-ngAEC6OCMeIZdj7Vd3G7MN22_AXvBBSMdovw&vprv=1&svpuc=1&mime=video%2Fwebm&gir=yes&clen=150553936&dur=1121.840&lmt=1692169289266051&mt=1692207200&fvip=5&keepalive=yes&fexp=24007246&beids=24350018&c=ANDROID&txp=6309224&sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cpcm2%2Cspc%2Cvprv%2Csvpuc%2Cmime%2Cgir%2Cclen%2Cdur%2Clmt&sig=AOq0QJ8wRgIhAKA0aSyme84yR_RFtdN76eyEtNwUdNqUkm54AnF-4OmMAiEA-BwD6MTf05O4jEQ87mU5WvhWCQaCCvQWk-LzHZDbrrw%3D&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpcm2cms%2Cpl%2Cinitcwndbps&lsig=AG3C_xAwRQIhALCWrVjRNAuuzFsei9GWoYRjw0LHdvI21ksG7_v0JNYBAiBvtulRj_xEjqVsQFpgfwQvg9OUxgDmpypAGJbw-3R8WQ%3D%3D"
ERROR: unable to download video data: HTTP Error 403: Forbidden
Traceback (most recent call last):
  File "yt_dlp/YoutubeDL.py", line 3213, in process_info
  File "yt_dlp/YoutubeDL.py", line 2954, in dl
  File "yt_dlp/downloader/common.py", line 444, in download
  File "yt_dlp/downloader/http.py", line 376, in real_download
  File "yt_dlp/downloader/http.py", line 129, in establish_connection
  File "yt_dlp/YoutubeDL.py", line 3726, in urlopen
  File "urllib/request.py", line 525, in open
  File "urllib/request.py", line 634, in http_response
  File "urllib/request.py", line 563, in error
  File "urllib/request.py", line 496, in _call_chain
  File "urllib/request.py", line 643, in http_error_default
urllib.error.HTTPError: HTTP Error 403: Forbidden
```

**Comment by bashonly** [comment]
> [debug] yt-dlp version 2023.02.17 [a0a7c01] (linux_dir)

Solution: update yt-dlp

see pinned issues

**Comment by report-bugs** [comment]
> > [debug] yt-dlp version 2023.02.17 [[a0a7c01](https://github.com/yt-dlp/yt-dlp/commit/a0a7c0154252900b7b154898744b698624d92b2a)] (linux_dir)
> 
> Solution: update yt-dlp
> 
> see pinned issues

I use last version: 2023-07-06. I updated yesterday to last known version.

**Comment by bashonly** [comment]
Look at your log; that is not the version that you are running

`which yt-dlp` will tell you the path that of the binary that is executed when you run `yt-dlp` in your terminal. Make sure it is the path to the new version

**Comment by report-bugs** [comment]
Now, I three times downloaded yt-dlp_linux from [here](https://github.com/yt-dlp/yt-dlp/releases)... Why after unpacking the date is Ju**n** 2023 instead of Ju**l** 2023? 

Where can I download zip package for Ubuntu linux?

**Comment by pukkandan** [comment]
`yt-dlp_linux` is a standalone executable and not meant to be unpacked. Just run it. Or see https://github.com/yt-dlp/yt-dlp#release-files for other options

**Comment by report-bugs** [comment]
> `yt-dlp_linux` is a standalone executable and not meant to be unpacked. Just run it. Or see https://github.com/yt-dlp/yt-dlp#release-files for other options

Thanks for help. It works. 
```
[debug] Command-line config: ['-f', '18', '-vU', 'https://www.youtube.com/watch?v=4CtBuDfO4Vc']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2023.07.06 [b532a3481] (linux_exe)
[debug] Python 3.10.12 (CPython x86_64 64bit) - Linux-5.4.0-156-generic-x86_64-with-glibc2.31 (OpenSSL 3.1.1 30 May 2023, glibc 2.31)
[debug] exe versions: ffmpeg 4.2.7, ffprobe 4.2.7, rtmpdump 2.4
[debug] Optional libraries: Cryptodome-3.18.0, brotli-1.0.9, certifi-2023.05.07, mutagen-1.46.0, sqlite3-2.6.0, websockets-11.0.3
[debug] Proxy map: {}
[debug] Loaded 1855 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Available version: stable@2023.07.06, Current version: stable@2023.07.06
Current Build Hash: 0556fff82db6c942a5dfbb9583e2896013847b28b95186b3800f730a8d5e0c0a
yt-dlp is up to date (stable@2023.07.06)
[youtube] Extracting URL: https://www.youtube.com/watch?v=4CtBuDfO4Vc
[youtube] 4CtBuDfO4Vc: Downloading webpage
WARNING: [youtube] Unable to download webpage: HTTP Error 429: Too Many Requests
[youtube] 4CtBuDfO4Vc: Downloading ios player API JSON
[youtube] 4CtBuDfO4Vc: Downloading android player API JSON
[youtube] 4CtBuDfO4Vc: Downloading iframe API JS
WARNING: [youtube] Unable to download webpage: HTTP Error 429: Too Many Requests
[youtube] 4CtBuDfO4Vc: Downloading web player API JSON
[youtube] 4CtBuDfO4Vc: Downloading m3u8 information
[youtube] 4CtBuDfO4Vc: Downloading initial data API JSON
[debug] Sort order given by extractor: quality, res, fps, hdr:12, source, vcodec:vp9.2, channels, acodec, lang, proto
[debug] Formats sorted by: hasvid, ie_pref, quality, res, fps, hdr:12(7), source, vcodec:vp9.2(10), channels, acodec, lang, proto, size, br, asr, vext, aext, hasaud, id
[info] 4CtBuDfO4Vc: Downloading 1 format(s): 18
[debug] Invoking http downloader on "https://rr2---sn-ntnxax8xo-cxge.googlevideo.com/videoplayback?expire=1692232782&ei=7hfdZPm9EvWt6dsPhbqDeA&ip=185.220.101.4&id=o-ABk-OEKZV1Ktt2GcMY8SGtU31evzQ1CHMi6px0ycyQW7&itag=18&source=youtube&requiressl=yes&mh=2h&mm=31%2C29&mn=sn-ntnxax8xo-cxge%2Csn-4g5edn6r&ms=au%2Crdu&mv=m&mvi=2&pl=26&initcwndbps=470000&spc=UWF9fy3MLQW2aDlnNlOrsL30iFHL5fw&vprv=1&svpuc=1&mime=video%2Fmp4&gir=yes&clen=69275566&ratebypass=yes&dur=1121.895&lmt=1692208976070824&mt=1692210795&fvip=2&fexp=24007246%2C51000024&beids=24350017&c=ANDROID&txp=5438434&sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cspc%2Cvprv%2Csvpuc%2Cmime%2Cgir%2Cclen%2Cratebypass%2Cdur%2Clmt&sig=AOq0QJ8wRQIhAMU-_jGMlF3Q6wwwASNoRM3oFbYltZIlHMiZBhSFJHQZAiAFOdj-xITeN4Qu1uTwz8VOKttUtiwRktclAfhhdGg6rQ%3D%3D&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lsig=AG3C_xAwRQIhANXfSY4yiMi9g9LRkNJcYwSY-_37WmwICWScsRReb3HiAiAVxNW_QYGOGtFajJ4axYQU4y8STXzJColKlEbSfMRMvA%3D%3D"
[download] Destination: Najwyższy Alarm w Europie： Rosjanie Przekroczyli Granice NATO [4CtBuDfO4Vc].mp4
[download]   2.1% of   66.07MiB at  185.91KiB/s ETA 05:56
```
